src/main/scala/com/cognite/sdk/scala/v1/resources/threeD.scala

```diff
@@ -108,6 +108,39 @@ class ThreeDRevisions[F[_]](val requestSession: RequestSession[F], modelId: Long
       }
   }
 
+  def listThreeDNodes(revisionId: Long): F[Seq[ThreeDNode]] = {
```

Comment:
From https://docs.cognite.com/api/v1/#operation/get3DNodes
> This operation supports pagination.
so this should return a `Stream` (an `fs2.Stream`), just like
https://github.com/cognitedata/cognite-sdk-scala/blob/715b8a3a1f1be816001e6cf48d9ee461d6df27dc/src/main/scala/com/cognite/sdk/scala/common/ReadableResource.scala#L58

We might want to make this into its own resource type, so we can do `client.threeDNodes(model, revision)` and `client.threeDNodes(model, revision).ancestors(nodeId)`

Resources implementing `Readable` get this by implementing
https://github.com/cognitedata/cognite-sdk-scala/blob/715b8a3a1f1be816001e6cf48d9ee461d6df27dc/src/main/scala/com/cognite/sdk/scala/common/ReadableResource.scala#L17

If not, you can get a `fs2.Stream` by using
https://github.com/cognitedata/cognite-sdk-scala/blob/715b8a3a1f1be816001e6cf48d9ee461d6df27dc/src/main/scala/com/cognite/sdk/scala/common/ReadableResource.scala#L92
directly, which needs a `get` function of three arguments: an optional cursor, an optional limit, and an optional partition (which is exactly what `readWithCursor` is).

Reply:
Or even client.threeDRevisions(modelId).nodes(revision).list()` and `client.threeDRevisions(modelId).nodes(revision).ancestors(nodeId)`